setup.py

```diff
@@ -87,19 +69,27 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     return py
 
 
-# https://packaging.python.org/discussions/install-requires-vs-requirements /
-# keep the meta-data here for simplicity in reading this file... it's not obvious
-# what happens and to non-engineers they won't know to look in init ...
-# the goal of the project is simplicity for researchers, don't want to add too much
-# engineer specific practices
 if __name__ == "__main__":
-    _SETUP_TOOLS = _load_py_module(name="setup_tools", location=os.path.join(".actions", "setup_tools.py"))
-
-    if _PACKAGE_NAME not in _PACKAGE_MAPPING:  # install everything
-        _SETUP_TOOLS._load_aggregate_requirements(_PATH_REQUIRE, _FREEZE_REQUIREMENTS)
```

Comment:
While we are here, could we add an error if `PACKAGE_NAME` is an invalid name. Ran into this issue already (e.g., lightning-lite vs lightning_lite) and it's annoying 😄

Reply:
That has been added already